src/vm/functions/mod.rs

```diff
@@ -200,7 +200,8 @@ macro_rules! native_hash_func {
             let bytes = match input {
                 Value::Int(value) => Ok(value.to_le_bytes().to_vec()),
                 Value::UInt(value) => Ok(value.to_le_bytes().to_vec()),
-                Value::Buffer(value) => Ok(value.data),
+                Value::Sequence(SequenceData::Buffer(value)) => Ok(value.data),
+                Value::Sequence(SequenceData::String(CharType::ASCII(value))) => Ok(value.data),
```

Comment:
Should UTF-8 strings also be hashable?

Reply:
I think it should yes

Reply:
Hmm. Should ASCII strings be hashable in a first place?

Reply:
I don't think so -- I'd be happy leaving this a Buffers and Ints only for now, and then addressing it via https://github.com/clarity-lang/reference/issues/18

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1779/commits/79ae83ed550b61a6aac2aec8e9b2869d225fe182